Formula/squashfs.rb

```diff
@@ -12,8 +12,11 @@ class Squashfs < Formula
     sha256 "c31a330128c85dc7fb8b2bd320f9a0f00469eff571323a6bbde435fa98496104" => :mavericks
   end
 
+  option "with-lz4", "Build with lz4 support"
```

Comment:
This line isn't necessary; the `:optional` flag automatically generates the option.